core/domain/activity_domain.py

```diff
@@ -28,17 +28,30 @@ class ActivityReference(object):
     """
```

Comment:
@seanlip Then this will be the case in each class as argument to \__init__ will be same as that of attribute of a class. I guess I understood the point. Thank you.